actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -603,28 +600,28 @@ public void onRead(final HandlerContext ctx, final Object msg) throws Exception
         }
     }
 
-    private void onInvocation(final HandlerContext ctx, final Invocation invocation)
+    private Task onInvocation(final HandlerContext ctx, final Invocation invocation)
```

Comment:
Without this onInvocation essentially becomes a blocking invocation when looking up the cached address.